logstash-core/lib/logstash/pipeline.rb

```diff
@@ -38,6 +38,7 @@ def self.validate_config(config_str, settings = {})
     begin
       # There should be a better way to test this
       self.new(config_str, settings)
+      true
     rescue => e
       e.message
```

Comment:
I think that returning a `string` or `false` is keeping the contract of returning something `truthy` because a string is `truthy`